tests/publiccloud/prepare_tools.pm

```diff
@@ -88,8 +79,7 @@ sub run {
     record_info('IPA', script_output('ipa --version'));
 
     # Install Terraform from repo
-    my $tfm_repo = get_var('TERRAFORM_REPO', 'https://download.opensuse.org/repositories/systemsmanagement:/terraform/' . generate_version() . '/systemsmanagement:terraform.repo');
-    zypper_call('ar ' . $tfm_repo);
+    zypper_call('ar https://download.opensuse.org/repositories/systemsmanagement:/terraform/SLE_15/systemsmanagement:terraform.repo');
```

Comment:
Is there a reason why you removed that var `TERRAFORM_REPO` ?

Reply:
It was discussed here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7842#discussion_r301688089

Reply:
yeah @jlausuch - I just don't like Test suite variables . @cfconrad so main disadvantage that you can't track changes in variable value , but if you think better to have it I can revert this change

Reply:
No keep it as it is.